cartfunctions.php

```diff
@@ -79,7 +78,28 @@ function DeleteCart($cart)
         if (array_key_exists($id, $cart)) {
             $cart[$id] = null;
             $_SESSION["cart"] = $cart;
+            print(" <p  class='AddCartMessage' >  Winkelmand geleegd </a> </p>");
+        }
+    }
+}
+
+function AddToCart($ID){
+    if (isset($_POST["submit"])) {
+        $stockItemID = $_POST["stockItemID"];
+        if (isset($_SESSION['cart'])) {  // controleren of winkelmandje al bestaat
+            $cart = $_SESSION['cart']; // zo ja: ophalen
```

Comment:
Je geeft $ID mee aan de functie maar vervolgens doe je in de functie niks met $ID